### DISCUSSION TITLE
Huawei App shows solar power of all my inverters, OpenAPI and FusionSolar seem not to have this information

### DISCUSSION THREAD

**Discussion by Tobias-Georg** [tijsverkoyen/HomeAssistant-FusionSolar#166] [Q&A]
Hi,
I have two solar installations, 
1. A SUN2000 4.6KLT
2. A SMA inverter
The solar power of the SMA inverter is measured with a 1-phase meter that is connected to the SUN2000. 
I am using FusionSolar to access Huawei's OpenAPI. The list of devices does not show a second inverter or a second meter.  However, the Huawei App shows correctly the sum of both inverters in the diagram on the starting page (showing the solar power coming from the "roof".
For an integration with my car charger, I'd like to get the total solar energy like the Huawei mobile app shows.
Any idea how I can get it?

**Comment by tijsverkoyen** [comment]
What do you mean with:
> I am using FusionSolar to access Huawei's OpenAPI. 

Do you mean this integration? Or the API directly?

> The list of devices does not show a second inverter or a second meter. 
Do you mean the devices in Home Assistant

**Comment by Tobias-Georg** [comment]
I am using Huawei's OpenAPI with some Python script that I downloaded years ago. Sorry that I confused it with your FusionSolar integration, it was named similarly.

When I use https://eu5.fusionsolar.huawei.com/thirdData/getDevList I get
{'devDn': 'NE=...', 'devName': 'Dongle-1', 'devTypeId': 62, ...}
{'devDn': 'NE=...', 'devName': 'Battery-1', 'devTypeId': 39, ...}
{'devDn': 'NE=...', 'devName': 'Inverter-1', 'devTypeId': 38, ...}
{'devDn': 'NE=...', 'devName': 'Meter-1', 'devTypeId': 47, ...}
With Meter-1 being the 3-phase meter that is used for the battery operation (measuring the current into the grid, so that the battery can control it to zero.)
Thus, if I understand well, I have not any info for the overall solar production, see my initial request.

**Comment by tijsverkoyen** [comment]
If the devlist does not include the meter you will not be able to get any data.
But I would ask your installer to check if you have the needed rights for that specific meter.
